docs/root/start/install.rst

```diff
@@ -201,6 +208,12 @@ The following table shows the available Docker images
      - |DOCKER_IMAGE_TAG_NAME|
      -
      -
+   * - `envoyproxy/envoy-distroless <https://hub.docker.com/r/envoyproxy/envoy-distroless/tags/>`_
+     - Release binary with symbols stripped on top of a distroless base.
+     -
+     -
+     - See Docker Hub
```

Comment:
please also make these match the alpine images (ie amd64 only and the correct version info)

Reply:
Could I use `DOCKER_IMAGE_TAG_NAME` directly?

Reply:
that is correct